src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -365,17 +378,38 @@ const LandingPage = () => {
                   data-testid="go-to-mhv-download-records"
                   className="vads-u-margin-bottom--2"
                 >
-                  <Link
-                    to="/download"
+                  <va-link-action
+                    href="/my-health/medical-records/download"
                     className="vads-c-action-link--blue"
                     data-testid="go-to-download-mr-reports"
-                    onClick={() => {
+                    text={MEDICAL_RECORDS_DOWNLOAD_LABEL}
+                    onClick={event => {
+                      event.preventDefault();
+                      history.push('/download');
                       sendDataDogAction(MEDICAL_RECORDS_DOWNLOAD_LABEL);
                     }}
-                  >
-                    {MEDICAL_RECORDS_DOWNLOAD_LABEL}
-                  </Link>
+                  />
+                </p>
+              </section>
+              <section className="vads-u-padding-bottom--3">
```

Comment:
You are adding this whole section, so my guess is you need to add something to the E2E tests to make sure this is there.